src/ice_masks.jl

```diff
@@ -56,15 +144,13 @@ function get_ice_masks(
     ice_mask = BitMatrix(zeros(Bool, sz))
     binarized_tiling = zeros(Int, sz)
 
-    fc_landmasked = apply_landmask(falsecolor_image, landmask)
+    fc_landmasked = apply_landmask(falsecolor_image, .!landmask)
 
-    Threads.@threads for tile in tiles
-        #  Conditionally update binarized_tiling as its not used in some workflows
-        if binarize
-            binarized_tiling[tile...] .= imbinarize(morph_residue[tile...])
-        end
-
-        morph_residue_seglabels = kmeans_segmentation(Gray.(morph_residue[tile...] / 255))
+    # Threads.@threads
```

Comment:
Should this be uncommented to utilize the threads?

Reply:
Sometimes it would crash, not sure why. Maybe I was running out of memory. I just decided to leave it commented for now to ensure it'd always work.

Reply:
#524